HW3/task.py

```diff
@@ -0,0 +1,28 @@
+
+# task 1
+print(("I love Python ") * 42)
```

Comment:
nit: brackets could be omitted
```suggestion
print("I love Python " * 42)
```